WordPress/src/main/res/layout/fragment_login_no_sites_empty_view.xml

```diff
@@ -0,0 +1,126 @@
+            app:constraint_referenced_ids="image_avatar,text_username, text_displayname"
+            tools:visibility="visible"/>
+
+        <ImageView
+            android:id="@+id/image_avatar"
+            android:layout_width="@dimen/login_empty_view_avatar_width"
+            android:layout_height="@dimen/login_empty_view_avatar_height"
+            android:layout_marginBottom="@dimen/login_empty_view_avatar_margin_bottom"
+            android:contentDescription="@string/login_empty_view_avatar_content_description"
+            app:layout_constraintBottom_toTopOf="@+id/text_displayname"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:srcCompat="@drawable/ic_user_white_24dp" />
+
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/text_username"
+            style="@style/Login.EmptyView.TextView.Username"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:paddingBottom="@dimen/margin_extra_medium_large"
+            app:layout_constraintBottom_toTopOf="@+id/login_empty_view_illustration"
+            app:layout_constraintStart_toStartOf="@+id/text_displayname"
+            app:layout_constraintTop_toBottomOf="@+id/text_displayname"
+            tools:text="\@droidtester2021" />
+
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/text_displayname"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:layout_constraintBottom_toTopOf="@+id/text_username"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/image_avatar"
+            tools:text="Droid Tester"
+            style="@style/Login.EmptyView.TextView.DisplayName"/>
+
+        <ImageView
+            android:id="@+id/login_empty_view_illustration"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:contentDescription="@string/login_no_jetpack_sites"
+            android:gravity="center_horizontal"
+            android:padding="@dimen/margin_extra_large"
+            android:src="@drawable/img_jetpack_empty_state"
+            app:layout_constraintBottom_toTopOf="@+id/login_error_message_title"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/login_empty_view_user_section"/>
+
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/login_error_message_title"
+            style="@style/ActionableEmptyStateTitle"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/margin_medium"
```

Comment:
Question (❓): The design specifies that this is `6dp` instead. Am I missing something?

Reply:
@ParaskP7  - there were no specs for this view ... so this is based this off of Woo and then adjusted to match what Chris and I discussed. Where are you seeing a spec sheet for this?